vdr/types/common.go

```diff
@@ -52,6 +52,8 @@ type DocumentMetadata struct {
 	Updated *time.Time `json:"updated,omitempty"`
 	// Hash of DID document bytes. Is equal to payloadHash in network layer.
 	Hash hash.SHA256Hash `json:"hash"`
+	// Transactions points to the transaction(s) that created this DID Document. If multiple transactions are listed, the DID Document is conflicted
+	Transactions []hash.SHA256Hash `json:"txs"`
```

Comment:
I find "Transactions" very non-descriptive; the name doesn't explain what the meaning of the field is, just what it contains.

Reply:
Called it `SourceTransactions`